doc/API.md

```diff
@@ -28,17 +32,379 @@ struct {
+an output of the transaction. When looking at transactions, it should be noted
+that a 'Siacoin Input' actually represents outgoing siacoins, as they are an
+input to the transaction; they are being spent. A 'Siacoin Output' represents
+incoming coins, because the output is being created by the transaction and made
+available to the wallet.
+
+'OutputID' is the id of the output. OutputIDs will always be unique.
+
+'RelatedAddress' is the address that is affected. For inputs (outgoing money),
+the related address is usually not important because the wallet arbitrarily
+selects which addresses will fund a transaction. For outputs (incoming money),
+the related address field can be used to determine who has sent money to the
+wallet.
+
+'Value' indicates how much money has been moved in the input or output.
+
+#### /wallet/history/$(addr) [GET]
+
+Function: Return all of the transaction related to a specific address.
+
+Parameters: none
+
+Response:
+```
+struct {
+	transactions []WalletTransaction
+}
+```
+'transactions' is a list of 'WalletTransactions' that affect the input address.
+See the documentation for '/wallet/history' for more information.
+
+#### /wallet/seed [GET]
+
+Function: Return the seed that is being used to generate addresses. This seed
+can be used to derive secret keys, and must be kept safe. This call is
+unavailable when the wallet is locked or closed.
+
+Parameters:
+```
+struct {
+	dictionary string
+}
+```
+'dictionary' is the name of the dicitionary that should be used when encoding
+the seed.
+
+Response:
+```
+struct {
+	primarySeed        string
+	addressesRemaining int
+	allSeeds           []string
+}
+```
+'primarySeed' is the seed that is actively being used to generate new addresses
+for the wallet.
+
+'addressesRemaining' is the number of addresses that remain in the primary seed
+until exhaustion has been reached and no more addresses will be generated.
+
+'allSeeds' is a list of all seeds that the wallet references when scanning the
+blockchain for outputs. The wallet is able to spend any output generated by any
+of the seeds, however only the primary seed is being used to generate new
+addresses.
+
+A seed is an encoded version of a 128 bit random seed. The output is 15 words
+chosen from a small dictionary as indicated by the input. The most common
+choice for the dictionary is going to be 'english'. The underlying seed is the
+same no matter what dictionary is used for the encoding. The encoding also
+contains a small checksum of the seed, to help catch simple mistakes when
+copying. The library
+[entropy-mnemonics](https://github.com/NebulousLabs/entropy-mnemonics) is used
+when encoding.
+
+#### /wallet/seed [PUT]
+
+Function: Get a new address from the wallet generated by the primary seed. An
+error will be returned if the seed has been exhausted (there is a limit on the
+number of addresses that can be generated from one seed), or if the wallet is
+locked/closed.
```

Comment:
/wallet/seed POST does something completely different.

/wallet/seed PUT grabs the next address from the existing seed, essentially advancing the seed a step.

/wallet/seed POST replaces the current seed with a completely new seed.